README.md

```diff
@@ -3,19 +3,23 @@
 1. Create a new, empty folder called opensource/ in your computer's home folder.
 2. Navigate to it (`cd opensource`), then fork and clone the ams repo so that it gets downloaded into opensource/ams.
 3. Navigate to this folder (`cd ams`) and run `git remote add upstream https://github.com/codethechange-niec/ams.git` (or, if you're using SSH, use `git@github.com:codethechange-niec/ams.git`) to add an upstream remote that's linked to ams's main repository. (This will be necessary when submitting PRs later.)
-4. Create a new database(MySQL) by following these commands:
+4. Create a new database(MySQL) and import ams.sql to it by following these commands:
   1. `mysql -u root -p` (Enter root password when prompted)
-  2. `CREATE USER 'ams_user'@'localhost' IDENTIFIED BY 'ams@123';`
```

Comment:
I think we should create a new user for this app. I don't think if it is a good idea to use the root user. @shubhamjadon what do you think?

Reply:
Creating a new user or using an old user is the developer's concern, he imports just a copy of main database. When this app deployed on a remote database server then we will create users and set necessary privileges and give to programmers.

Reply:
But I think the way I am suggesting, the new developers will learn how to create a new user and set privileges. It will be a learning experience for them.

Reply:
That's why I provided the WhatsApp grp.
If an experienced developer wants to contribute then he can use DB parameters whatever he wants.
If a newbie comes then he can ask on the group which not only increases group conversation but also the developer involvement in open source conversation.

Reply:
I think no one has that much to repeat same thing again and again.

Reply:
Let's wait for @shubhamjadon comment.

Reply:
As main motive of this repo is to teach so we should create as many opportunities for learners as possible to learn. So, that's why i think it will be better to create a new user for this app. And also @mohitsingh1930 i think it would be better to keep all things on github only (including support) as i do not think whatsapp grp is a good place for discussion with newbies. For starting we can use only github issues for handling newbies queries and with further growth we can use slack or irc channels. 
@Hudda @mohitsingh1930 share your views regarding later too.

Reply:
@shubhamjadon agreed.

Reply:
@mohitsingh1930 ptal and make changes as required.

Reply:
Ok i agreed with @shubhamjadon.
I will make changes to Readme and then push it.

Reply:
@Hudda PTAL, I re-added create user MySQL instructions in README.md.